lighthouse-core/runner.js

```diff
@@ -115,7 +113,8 @@ class Runner {
       const resultsById = {};
       for (const audit of auditResults) resultsById[audit.name] = audit;
 
-      let reportCategories;
+      /** @type {Array<LH.Result.Category>} */
+      let reportCategories = [];
```

Comment:
part of the discussed change to default to having an empty `categories` array, even if none in the config (we could maybe start defaulting to `[]` in the config, but we'll see)